psi4/driver/procrouting/sapt/fisapt_proc.py

```diff
@@ -91,7 +91,26 @@ def fisapt_compute_energy(self):
         self.find()
         core.timer_off("FISAPT:FSAPT:ind")
         core.timer_on("FISAPT:FSAPT:disp")
-        self.fdisp()
+        if core.get_option("FISAPT", "FISAPT_DO_FSAPT_DISP"):
+            core.timer_on("FISAPT:FSAPT:disp")
+            self.fdisp()
+            core.timer_off("FISAPT:FSAPT:disp")
+            self.fdrop()
+        else:
```

Comment:
comment out this block please. let's get only what they need in. if they ran with fisapt_do_fsapt_disp=False, then ran a `mol.run_dftd3(... d3mbj)`, I could see someone getting confused over what empirical dispersion to believe.